### PR TITLE
Add recipe for idaes-pse

### PR DESCRIPTION
**Pull request by lbianchi-lbl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by lbianchi-lbl** [comment]
@conda-forge-admin please ping team @conda-forge/help-python

This is ready for review.

File: recipes/idaes-pse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# idaes_pse/meta.yaml
+{% set name = "idaes-pse" %}
+{% set version = "2.6.0" %}
+{% set python_min = "3.9" %}
```
Comment (xhochy) [inline recipes/idaes-pse/meta.yaml:4]: Not needed anymore.
```suggestion
```

File: recipes/idaes-pse/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# idaes_pse/meta.yaml
```
Comment (xhochy) [inline recipes/idaes-pse/meta.yaml:1]:
```suggestion
```

File: recipes/idaes-pse/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - idaes = idaes.commands.base:command_base
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-cache-dir --ignore-installed"
+
+requirements:
+  host:
+    - python {{ python_min }}
```
Comment (xhochy) [inline recipes/idaes-pse/meta.yaml:24]: Here are way too much things listed. There should only be the things that are required to build the package, e.g. `setuptools`.

File: recipes/idaes-pse/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    - sympy
+
+test:
+  requires:
+    - python {{ python_min }}
+  imports:
+    - idaes
+    - idaes.core
+    - idaes.core.base
+  commands:
```
Comment (xhochy) [inline recipes/idaes-pse/meta.yaml:60]:
```suggestion
  commands:
    - pip check
```

File: recipes/idaes-pse/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    - pint
+    - pip
+    - pyomo >=6.8.0
+    - pytest
+    - pyyaml
+    - scipy
+    - sympy
+
+test:
+  requires:
```
Comment (xhochy) [inline recipes/idaes-pse/meta.yaml:54]:
```suggestion
  requires:
    - pip
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/idaes-pse/meta.yaml (hunk 3 above)
Comment (lbianchi-lbl) [reply recipes/idaes-pse/meta.yaml:22]: Thanks, I see what you mean. Done in ed160f7b338e34180b11e19210c8036f10c231af

File: recipes/idaes-pse/meta.yaml (hunk 5 above)
Comment (lbianchi-lbl) [reply recipes/idaes-pse/meta.yaml:41]: Done in b0f407092ded4276007e6d57befb2a410bd7f0d4

File: recipes/idaes-pse/meta.yaml (hunk 4 above)
Comment (lbianchi-lbl) [reply recipes/idaes-pse/meta.yaml:48]: Done in f98fa230267239c4e8119ba5a66fe432030d8c7d

**Comment by lbianchi-lbl** [comment]
Thanks a lot for the feedback @xhochy. I believe I have addressed all your comments, but let me know if that's not the case.

**Pull request merged**